lib/echo/echo_client.rb

```diff
@@ -59,7 +59,7 @@ def get_availability_events
     end
 
     def get_order_information(item_ids, token)
-      get('/echo-rest/order_information.json', {catalog_item_id: item_ids}, token_header(token))
+      post('/echo-rest/order_information.json', options_to_granule_query({catalog_item_id: item_ids}).to_query, token_header(token).merge('Content-Type' => 'application/x-www-form-urlencoded'))
```

Comment:
Why are you calling options_to_granule_query? I don't think it alters anything, and this is not a granule query, it's an order information query.